SysCache2/NHibernate.Caches.SysCache2/NHibernate.Caches.SysCache2.csproj

```diff
@@ -1,35 +1,36 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../NHibernate.Caches.props" />
-  <PropertyGroup>
-    <Product>NHibernate.Caches.SysCache2</Product>
-    <Title>NHibernate.Caches.SysCache2</Title>
-    <Description>Cache provider for NHibernate using ASP.NET Cache object. Supports SqlCacheDependency, allowing SQL Server changes to invalidate the cache.</Description>
-    <TargetFramework>net461</TargetFramework>
-    <SignAssembly>True</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\..\NHibernate.Caches.snk</AssemblyOriginatorKeyFile>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageReleaseNotes>* Improvement
-    * #60 - Update package format for symbols, license and SourceLink</PackageReleaseNotes>
-  </PropertyGroup>
-  <ItemGroup>
-    <None Include="..\..\NHibernate.Caches.snk" Link="NHibernate.snk" />
-    <None Include="..\default.build" Link="default.build" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" PrivateAssets="all" />
-    <PackageReference Include="NHibernate" Version="5.2.0" />
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Transactions" />
-    <Reference Include="System.Web" />
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="../../readme.md">
-      <PackagePath>./NHibernate.Caches.readme.md</PackagePath>
-    </Content>
-    <Content Include="../../LICENSE.txt">
-      <PackagePath>./NHibernate.Caches.license.txt</PackagePath>
-    </Content>
-  </ItemGroup>
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../NHibernate.Caches.props" />
+  <PropertyGroup>
+    <Product>NHibernate.Caches.SysCache2</Product>
+    <Title>NHibernate.Caches.SysCache2</Title>
+    <Description>Cache provider for NHibernate using ASP.NET Cache object. Supports SqlCacheDependency, allowing SQL Server changes to invalidate the cache.</Description>
+    <TargetFramework>net461</TargetFramework>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\NHibernate.Caches.snk</AssemblyOriginatorKeyFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageReleaseNotes>* Improvement
+    * #68 - Add static region configuration
```

Comment:
Why is the entire file marked as changed, apart from the added line nothing else seem to be changed?

Reply:
No idea. This looks like a GitHub glitch. In Git history (viewed with Rider), only the changed line is shown.

Reply:
Dodging the trouble by dropping the file then editing it through the web interface works. I have done the same for buildcommon.xml.
Still I suspect the glitch will be there again next time they are changed.